pkgs/tools/package-management/disnix/dydisnix/default.nix

```diff
@@ -1,18 +1,18 @@
-{ lib, stdenv, fetchFromGitHub, autoconf, automake, libtool , pkg-config, glib, libxml2, libxslt, getopt, libiconv, gettext, nix, disnix, libnixxml }:
+{ lib, stdenv, fetchFromGitHub, autoconf, automake, libtool , pkg-config, glib, libxml2, libxslt, getopt, libiconv, gettext, nix, disnix }:
 
 stdenv.mkDerivation rec {
-  version="2020-07-04";
+  version="2020-11-02";
```

Comment:
The version number should be prefixed with ``unstable-`` and also the file could get some formatting with nixpkgs-fmt.

Reply:
I didn't know that this is a requirement. I simply just changed what was already there.

I happen to be the author of dydisnix, but I'm not the maintainer of the package. It is @tomberek who packaged it and apparently has a use for it. I only use in experimental setups and typically deploy it directly from the Nix expressions in the development repository.

Reply:
I can of course tidy up the expression, no problem :)